common.lic

```diff
@@ -192,8 +192,14 @@ module DRC
 
   def stow_hands
     pause 1
-    bput('stow right', 'You put') if checkright
-    bput('stow left', 'You put') if checkleft
+    stow_hand('right') if checkright
+    stow_hand('left') if checkleft
+  end
+
+  def stow_hand(hand)
+    braid_regex = /The braided (.+) is too long/
+    result = bput("stow #{hand}", 'You put', braid_regex)
+    DRCI.dispose_trash(DRC.get_noun(Regexp.last_match(1))) if braid_regex.match(result)
```

Comment:
I did a test with a simple script that was just this:

```
custom_require.call(%w(dependency equipmanager drinfomon events spellmonitor common common-arcana common-items common-travel common-summoning common-money common-crafting textsubs roomnumbers))

#DRC.forage?('grass')
EquipmentManager.instance.empty_hands
```
And it errored without the DRC on the dispose_trash call.

As far as the circular dependency, I can't say. I don't know enough to fully understand dependencies in Ruby.